libcontainer/container_linux.go

```diff
@@ -2114,6 +2114,16 @@ func encodeIDMapping(idMap []configs.IDMap) ([]byte, error) {
 	return data.Bytes(), nil
 }
 
+func encodeTimeNs(timeOffsets specs.LinuxTimeOffset) ([]byte, error) {
+	data := bytes.NewBuffer(nil)
+	line := fmt.Sprintf("%d %d\n", timeOffsets.Secs, timeOffsets.Nanosecs)
+	if _, err := data.WriteString(line); err != nil {
+		return nil, err
+	}
+
+	return data.Bytes(), nil
```

Comment:
No need to use bytes buffer. You can just return `[]byte(str)`.

Reply:
As I understand, it's not required to have `bytes.NewBuffer(nil)` instead return the timeoffsets.
I've updated the changes in latest commit. Please verify.